frontend/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "build",
+    "module": "esnext",
+    "target": "ES2017",
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "sourceMap": true,
+    "allowJs": true,
+    "jsx": "react",
+    "rootDir": "src",
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "noEmitOnError": false,
+    "pretty": true,
+    "forceConsistentCasingInFileNames": false,
+    "noErrorTruncation": true,
+    "noImplicitReturns": true,
+    "noImplicitAny": false,
```

Comment:
This used to be `true`. Going back to `false` without an excellent reason would be a regression.

Reply:
because of this `substrate-telemetry/frontend/src/AfgHandling.ts
(16,10): 'receivedAuthoritySet', which lacks return-type annotation, implicitly has an 'any' return type.`

Reply:
and potentially other issues when you solve that. setting that to false is nothing major. if we put that back code will break. also deleting the yarn.lock from frontend will break it even on master. try it.

Reply:
even before all my big prs

Reply:
"Implicit any" issues are only showing up in ConnectionInput.tsx => specifying the 2 missing types
```
  function handleChange(e: React.ChangeEvent<HTMLInputElement>) {... }
  function handleSubmit(e: SyntheticEvent) { ... }
```